docs/.vitepress/config.ts

```diff
@@ -0,0 +1,66 @@
+const utils = require('./utils.ts');
+
+utils.copyFile('README.md', 'index.md');
+utils.copyFile('CONTRIBUTING.md', 'CONTRIBUTING.md');
```

Comment:
Previous implementation do this by symlink. But vitepress doesn't support it.
As a result, we need to copy the file instead.